src/style.css

```diff
@@ -1,3 +1,91 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&displ0ay=swap';
```

Comment:
The same font is added twice in the different places: here and in `<head>` of HTML. We don't need this, so please remove font addition from one of the places.